app/assets/javascripts/signin/login.coffee

```diff
@@ -5,7 +5,6 @@ class OX.Signin.Login
     new OX.Signin.Login(card) if card.length
 
   constructor: (@el) ->
-    console.log @el.find('a.trouble')
```

Comment:
Sigh, so it was me :(

I was searching for `puts` and `p` in the specs, didn't think to check JS.  Good catch @jpslav